lumicks/pylake/conftest.py

```diff
@@ -107,14 +108,37 @@ def configure_mpl():
         plt.close("all")
 
 
+def filename_hash(string_to_hash):
+    """Hash a filename for a test
+
+    Parameters
+    ----------
+    string_to_hash : str
+        String to return the hash of.
+
+    We use a base36 hash to ensure that we can safely store these hashes as filenames on all
+    operating systems. Base64 contains /, and has issues with case sensitivity on windows. Base36
+    is still shorter than a hexadecimal hash.
```

Comment:
You could use `urlsafe_b64encode()` which doesn't produce `/`: https://docs.python.org/3/library/base64.html#base64.urlsafe_b64encode

Also, take a look at this https://stackoverflow.com/questions/2510716/short-python-alphanumeric-hash-with-minimal-collisions. Something like this might work: 
```py
hash = hashlib.md5(string_to_hash.encode())
b64 = base64.urlsafe_b64encode(hash.digest())
return b65[:10]
```

Reply:
I suspect base64 would still lead to a lot of collisions since unlike URLs, I don't think the windows file system is case sensitive?

Reply:
Right, the case. Good thinking. Yeah, it would be good to add it to the comment.